packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendarHeaderAction.tsx

```diff
@@ -7,7 +7,7 @@ import { Button, ButtonProps, ButtonStylesProps } from '../Button/Button';
 
 export type DatepickerCalendarHeaderActionProps = ButtonProps & {
   /** What direction the action button should be pointing. */
-  direction?: 'previous' | 'next';
+  direction?: 'previous' | 'next' | string;
```

Comment:
_This change should not be required, check and update this message with a suggestion._

Reply:
Fixed